src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -266,7 +266,13 @@ describe('ReactElement', () => {
       React.createElement(Wrapper)
     );
 
-    expect(element._owner.getPublicInstance()).toBe(instance);
+    // If the element is owned by a Fiber, there is no methods
+    // We need to circumvent getPublicInstance() to introspect the owner
+    if (element._owner.stateNode) {
```

Comment:
Let's consistently check for `typeof instance.tag === 'number'` for Fiber-ness everywhere. It may not seem to make as much sense in individual cases but it's easier to have the same check everywhere in case we change the field names later or something like this.